opds2_import.py

```diff
@@ -71,7 +71,16 @@ def parse_manifest(self, manifest):
         """
         result = None
 
-        if isinstance(manifest, str):
+        if isinstance(manifest, bytes):
+            try:
+                input_stream = StringIO(manifest.decode("utf-8"))
```

Comment:
Should also be able to do this with `input_stream = BytesIO(manifest)` to avoid the explicit decode.